src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -10,8 +10,8 @@
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
```

Comment:
Should be INDEX_FIRST_PERSON instead: it's testing the original AB3 edit command